Cargo.toml

```diff
@@ -50,3 +53,8 @@ tracing = { version = ">=0.1.40", default-features = false }
 tracing-core = { version = ">=0.1.33", default-features = false } 
 tracing-subscriber = { version = "0.3", default-features = false }
 url = { version = "2.5", default-features = false }
+
+[patch.crates-io]
```

Comment:
can you put a comment on this file for future readers to understand the need for this?